diff --git a/parser/expat/lib/xmlparse.c b/parser/expat/lib/xmlparse.c
index 209f20e094dc..056974e1479d 100644
--- a/parser/expat/lib/xmlparse.c
+++ b/parser/expat/lib/xmlparse.c
@@ -839,7 +839,7 @@ generate_hash_secret_salt(XML_Parser parser)
 {
   unsigned long entropy;
   (void)parser;
-#if defined(HAVE_ARC4RANDOM_BUF) || defined(__CloudABI__)
+#if (defined(HAVE_ARC4RANDOM_BUF) || defined(__CloudABI__)) && !defined(ANDROID)
   (void)gather_time_entropy;
   arc4random_buf(&entropy, sizeof(entropy));
   return ENTROPY_DEBUG("arc4random_buf", entropy);
